.github/workflows/collector-test.yaml

```diff
@@ -0,0 +1,33 @@
+name: Test Collector Chart
+
+on:
+  pull_request:
+    paths: 
+    - 'charts/opentelemetry-collector/**'
+    branches:
+      - main
+
+jobs:
+  collector-test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Setup
+        uses: ./.github/actions/setup
+        with:
+          create-kind-cluster: "true"
+
+      - name: Run make check-examples
+        run: make check-examples
```

Comment:
Shouldn't this be part of lint workflow?

Reply:
Ya it should be. Forgot it wasn't collector specific.